docs/reference/commandline/dockerd.md

```diff
@@ -75,7 +75,8 @@ Options:
       --registry-mirror value                 Preferred Docker registry mirror (default [])
       --seccomp-profile value                 Path to seccomp profile
       --selinux-enabled                       Enable selinux support
-      --shutdown-timeout=15                   Set the shutdown timeout value in seconds
+      --shm-size int                          Set the default shm size for containers (default 67108864)
```

Comment:
Does that mean you can't use units with this new option? When using `docker run`, you can, so ideally it should be the same here. For instance: `--shm-size=128m`